CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
```

Comment:
Yes, that would be a good idea. :-)
Den 5 jan 2015 14:58 skrev "J-Shep" <notifications@github.com>:

> In CorsixTH/Lua/app.lua
> <https://github.com/CorsixTH/CorsixTH/pull/439#discussion-diff-22461388>:
>
> > @@ -313,6 +313,39 @@ function App:init()
> >    return true
> >  end
> >
> > +---
> > +-- Initialises a Lua DBGp client and
> > +-- connects it to a DBGp server so that
> > +-- it can debug lua scripts at runtime.
> > +-- @return error_message
>
> I could make a regular expression script to make all functions comments
> abide by the LDocGen function comment conventions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CorsixTH/CorsixTH/pull/439/files#r22461388>.
>